cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/QueryPlanTest.scala

```diff
@@ -209,23 +209,51 @@ class QueryPlanTest extends DocumentingTestBase with SoftReset {
 
   @Test def nodeByLabelScan() {
     profileQuery(
-      title = "Node by label scan",
-      text = """Using the label index, fetches all nodes with a specific label on them from the node label index.""".stripMargin,
+      title = "Node By Label Scan",
+      text = """The `NodeByLabelScan` operator fetches all nodes with a specific label from the node label index.""".stripMargin,
       queryText = """MATCH (person:Person) RETURN person""",
-      assertions = (p) => assertThat(p.executionPlanDescription().toString, containsString("LabelScan"))
+      assertions = (p) => assertThat(p.executionPlanDescription().toString, containsString("NodeByLabelScan"))
     )
   }
 
   @Test def nodeByIndexSeek() {
     profileQuery(
-      title = "Node index seek",
-      text = """Finds nodes using an index seek. The node variable and the index used is shown in the arguments of the operator.
-                |If the index is a unique index, the operator is called `NodeUniqueIndexSeek` instead.""".stripMargin,
+      title = "Node Index Seek",
+      text = """The `NodeIndexSeek`operator finds nodes using an index seek. The node variable and the index used is shown in the arguments of the operator.
+                |If the index is a unique index, the operator is instead called `NodeUniqueIndexSeek`.""".stripMargin,
       queryText = """MATCH (location:Location {name: 'Malmo'}) RETURN location""",
       assertions = (p) => assertThat(p.executionPlanDescription().toString, containsString("NodeIndexSeek"))
     )
   }
 
+  @Test def nodeByUniqueIndexSeek() {
+    profileQuery(
+      title = "Node Unique Index Seek",
+      text = """The `NodeUniqueIndexSeek` operator finds nodes using an index seek within a unique index. The node variable and the index used is shown in the arguments of the operator.
+               |If the index is not unique, the operator is instead called `NodeIndexSeek`.""".stripMargin,
+      queryText = """MATCH (t:Team {name: 'Malmo'}) RETURN t""",
+      assertions = (p) => assertThat(p.executionPlanDescription().toString, containsString("NodeUniqueIndexSeek"))
+    )
+  }
+
+  @Test def argument() {
+    profileQuery(
+      title = "Argument",
+      text = """The `Argument` operator indicates the variable to be used as an argument to the right-hand side of an <<query-plan-apply, Apply>> operator.""".stripMargin,
+      queryText = """MATCH (s:Person {name: 'me'}) MERGE (s)-[:FRIENDS_WITH]->(s)""",
+      assertions = (p) => assertThat(p.executionPlanDescription().toString, containsString("Argument"))
+    )
+  }
+
+  @Test def loadCSV() {
+    profileQuery(
+      title = "Load CSV",
+      text = """The `LoadCSV` operator is used when executing a query containing the <<query-load-csv, LOAD CSV>> clause.""".stripMargin,
```

Comment:
The LoadCSV operator loads data into the query from a CSV source. It is used whenever the LOAD CSV clause is used in a query.

Reply:
👍